backend/kernelCI_app/helpers/filters.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional
+from kernelCI_app.utils import UNKNOWN_STRING
+
+
+def is_build_invalid(build_valid: Optional[bool]) -> bool:
+    return build_valid is None or build_valid is False
+
+
+def is_known_issue(issue_id: Optional[str]) -> bool:
+    return issue_id is not None and issue_id is not UNKNOWN_STRING
+
+
+def is_issue_from_test(incident_test_id: Optional[str], is_known_issue: bool) -> bool:
+    return incident_test_id is not None or not is_known_issue
+
+
+def is_issue_filtered_out(issue_id: Optional[str], issue_filters):
```

Comment:
Is issue_filters here expected to be a list, set, dict or any combination of them (like `List | set`)? If so, I don't think there's a reason to not type hint this argument in this situation. You should also type the return value